Sources/Service/WalletAvailabilityService.swift

```diff
@@ -0,0 +1,24 @@
+import Foundation
+import Networking
+import UIKit
+
+
+public final class WalletAvailabilityService {
+    let defaults = UserDefaults.standard
+    
+    var walletVisibleToAll = AppEnvironment.walletVisibleToAll
+    var deeplinkAccepted = AppEnvironment.walletVisibleViaDeepLink
```

Comment:
similarly with these

Reply:
That's fair.. I thought it was useful more so for the WalletAvailabilityServiceTests. But will remove for now & try to find a better alternative for the tests 😅